src/Xamarin.Android.Build.Tasks/Tasks/CompileToDalvik.cs

```diff
@@ -115,7 +115,11 @@ protected override string GenerateCommandLineCommands ()
 			if (File.Exists (OptionalObfuscatedJarFile))
 				cmd.AppendFileNameIfNotNull (OptionalObfuscatedJarFile);
 			else {
-				cmd.AppendFileNameIfNotNull (ClassesOutputDirectory);
+				var zip = Path.GetFullPath (Path.Combine (ClassesOutputDirectory, "classes.zip"));
+				if (!File.Exists (zip)) {
+					Log.LogError ($"'{zip}' does not exist. Please rebuild the project.");
```

Comment:
`LogError()` doesn't throw an exception, so execution will continue after this point. Should an exception be thrown + caught in `Execute()` instead?